src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -346,7 +346,19 @@ public static void WriteStatsResult(JsonWriter jsonWriter, NuGetIndexSearcher se
             jsonWriter.WriteStartObject();
             WriteProperty(jsonWriter, "numDocs", searcher.IndexReader.NumDocs());
```

Comment:
I think this is an unrelated change to this PR, and simply doing the mapping is not enough. You need to check if the value is an object and recursively follow the object's properties to correctly marshal them into JSON. #won't fix